### ISSUE TITLE
Provide warning in web UI when JavaScript is turned off

### ISSUE THREAD

**Issue by brunchboy** [OpenLightingProject/ola#1046]
I just had a panic (helpfully resolved by kind folks on the IRC channel) thinking that my `olad` installation had gotten broken somehow, when it was because I left JavaScript turned off in my browser after trying to read news on a site which is plagued by aggressive ads. The pressure of preparing for a show prevented me from figuring this out on my own. It would be nice if the UI could help people in similar brain-impaired situations before panic sets in. I may be able to submit a PR myself to do this once I am past the pressure of preparing for my current show, but wanted to add this issue in case I forget, or in case someone else wants to get to it first.

**Comment by daveol** [comment]
Basically we want [<noscript>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/noscript) tags with the appropriate text, or maybe just an element that gets removed by a javascript call

**Comment by brunchboy** [comment]
Yup. I just investigated taking a crack at this because it seemed straightforward enough, but your Javascript is built using a complicated procedure and the [README](https://github.com/OpenLightingProject/ola/tree/master/javascript) is out of date and refers to resources which are no longer at the locations specified, and likely no longer available using `svn`. I am spending a few minutes to see if I can get past that, but need to get back to designing my show soon.

**Comment by brunchboy** [comment]
Yeah, I can’t even figure out how to compile it to test using a simple `<noscript>` approach. I found `mac.build.sh` but when I try it, I get:

    ➜ ola git:(master) ✗ mkdir /tmp/ola
    ➜ ola git:(master) ✗ ./mac_build.sh /tmp/ola/
    ./mac_build.sh: line 18: ./configure: No such file or directory
    make: *** No targets specified and no makefile found.  Stop.
    make: *** No rule to make target `check'.  Stop.
    Password:

I will leave this to someone who has practice making changes to OLA. :smile:

Oh, but wait! I found where the HTML lives in my installation from Homebrew, so I can test my changes there. I have come up with a simple solution that works and looks fine, so a PR is coming shortly.
